lib/spack/spack/binary_distribution.py

```diff
@@ -526,6 +526,7 @@ def make_package_relative(workdir, spec, allow_root):
         relocate.make_macho_binaries_relative(cur_path_names, orig_path_names,
                                               old_layout_root)
     if (spec.architecture.platform == 'linux' or
+        spec.architecture.platform == 'cray' or
```

Comment:
Lines 528-532 should probably be contracted to 
```
if (platform.system().lower() == 'linux'):
```

Reply:
I tried to clean this up by adding a `binary_formats` attribute to the platform.  @becker33 see what you think.

Reply:
So should I change to `platform.system().lower() == 'linux'` ?

Reply:
I would use the changes Todd made.

Reply:
Nevermind. I see what you did.